CMakePresets.json

```diff
@@ -40,7 +40,7 @@
     {
       "name": "windows-clang-x64",
       "binaryDir": "${sourceDir}/build",
-      "generator": "Visual Studio 16 2019",
+      "generator": "Visual Studio 17 2022",
```

Comment:
Is there a way to say "the latest Visual Studio available"?

Reply:
As far as I can tell from the [documentation](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#id14) all of the Visual Studio generators are for a specific version.